components/supervisor/pkg/supervisor/services.go

```diff
@@ -800,8 +807,23 @@ func (ss *ControlService) CreateSSHKeyPair(ctx context.Context, req *api.CreateS
 	}
 	ss.privateKey = string(generated.PrivateKey)
 	ss.publicKey = string(generated.PublicKey)
+
+	hostKey, err := os.ReadFile("/.supervisor/ssh/sshkey.pub")
```

Comment:
We get keys another place like this https://github.com/gitpod-io/gitpod/blob/7ef163a8e7eb433210a211a5d805ec1b67a475d3/components/supervisor/pkg/supervisor/ssh.go#L31 . It can be improved but no-block